### ISSUE TITLE
Error 1x1 Message: "Password:" "installer: Error - the package path specified was invalid: '/Users/atmb82/.patched-sur/InstallAssistant.pkg'."

### ISSUE THREAD

**Issue by atmb82** [BenSova/Patched-Sur#173]
**Describe the bug**
after inserting my password, I get this error when the patcher should download the Install Assistant Package

**To Reproduce**
1. start patcher
2. select "release"
3. select "update" (11.1)
4. insert password
5. error appears

**Expected behavior**
it should download and then create the bootable usb drive
![Schermata 2020-12-29 alle 10 53 05](https://user-images.githubusercontent.com/4113527/103275251-13d3d700-49c4-11eb-9db9-a01970837d36.png)

**Screenshots**
Error screen

**Desktop (please complete the following information):**
 - Upgrading from: Big Sur 11.0.1 (20B29)
- Upgrading to: Big Sur 11.1
 - Mac Model: iMac late 2013 27" i7-4771
 - Version v0.0.4 (14)

If there are any red boxes, click on them to copy the logs.
Here you are:
Error 1x1
Message: "Password:"
"installer: Error - the package path specified was invalid: '/Users/atmb82/.patched-sur/InstallAssistant.pkg'."

**Comment by RicardoSilva01** [comment]
Close the Patched Ser app, remove the .patched-sur folder completely and restart the app.

**Comment by BenSova** [comment]
Yeah, looks like your download just got corrupted. Just trying again is a simple fix a lot of times.

**Comment by phillgates** [comment]
"Close the Patched Ser app, remove the .patched-sur folder completely and restart the app." 

That only breaks the app and doesn't work, besides there seems to be an issue with the late 2013 (If you can call it that), I have the 21" model 2013 iMac and everything hardware wise is running without the patcher that is supposed to be used after install, and I mean everything from wifi to USB, even the SSD and Trim are all fine, so other than anything I am not aware off in the Kernel that the patcher modifies, then it seems that on the late 2013 there is no need to patch. I also had the warning issue with the after boot patcher.

There is an issue with software that I do get and that is under normal shutdown or restart an apple message appears during the process that states there was an issue that forced shutdown, which is obviously incorrect and it doesn't always happen, seems a minor bug, may be related to SIP, those are my observations.

**Comment by BenSova** [comment]
You are talking about a completely different thing. Those files are created by the pre-install app, so they obviously aren't required to run the pre-install app. You are referring to the post install app, that won't need those files after v0.1.0 is out.

**Comment by joycillin** [comment]
Dear BenSova,
I have the exact same problem! I've also tried deleting the .patched-sur folder completely and re-opened the installer. However the same problem re-occurs. I'm on the late 2012 MacBook pro retina 15'. Thank you.

**Comment by luchi101** [comment]
Error 1x1
Message: "Password:Error 2x1: Installer Not Found"
Output: "Welcome to Patched Sur! (v0.0.6)
This script *should only* be used through
the Patched Sur app unless being tested
for patcher development

Detecting Installer USB at /Volumes/Install macOS Big Sur [Beta]...
Installer USB was not detected.
Please be sure to not rename the USB before
running patch-usb.sh.
"

**Comment by BenSova** [comment]
Closing in support of #15
